tests/unittests/CoreText/CTFontTests.mm

```diff
@@ -46,35 +59,28 @@ virtual void TearDown() {
     CFRelease(propertyValue);
 }
 
-const NSString* c_copyrightName = @"© 2015 Microsoft Corporation. All Rights Reserved.";
-const NSString* c_familyName = @"Segoe UI";
+const NSString* c_familyName = @"Metadata Test";
 const NSString* c_subFamilyName = @"Regular";
 const NSString* c_styleName = @"Regular";
-const NSString* c_uniqueName = @"Segoe UI Regular";
-const NSString* c_fullName = @"Segoe UI";
-
-// version name for Segoe UI font is different on ARM than on x86 platform, so we are using different version names for both platforms.
-#ifdef _M_ARM
-const NSString* c_versionName = @"Version 5.53; sf";
-#else
-const NSString* c_versionName = @"Version 5.53";
-#endif
+const NSString* c_uniqueName = @"Metadata Test Regular";
+const NSString* c_fullName = @"Metadata Test";
 
-const NSString* c_postscriptName = @"SegoeUI";
-const NSString* c_trademarkName = @"Segoe is a trademark of the Microsoft group of companies.";
+const NSString* c_postscriptName = @"MetadataTest-Regular";
+const NSString* c_trademarkName = @"MetadataTest is not a trademark.";
 const NSString* c_manufacturerName = @"Microsoft Corporation";
 const NSString* c_designerName = nullptr;
 const NSString* c_descriptionName = nullptr;
-const NSString* c_vendorURLName = @"http://www.microsoft.com/typography/fonts/";
+const NSString* c_vendorURLName = @"https://developer.microsoft.com/en-us/windows/bridges/ios";
 const NSString* c_designerURLName = nullptr;
-const NSString* c_licenseURLName = @"http://www.microsoft.com/typography/fonts/";
+const NSString* c_licenseURLName = @"https://developer.microsoft.com/en-us/windows/bridges/ios";
 const NSString* c_sampleTextName = nullptr;
+const NSString* c_copyrightName = @"Copyright (c) Microsoft";
```

Comment:
aww, these moved around w.r.t. the true original. the diff could look so much prettier if Version was below fullName and copyright was above familyName! :P